### ISSUE TITLE
npm installation instructions

### ISSUE THREAD

**Issue by Jimadine** [artefactual/atom-docs#178]
I've noticed a few oddities relating to the `npm` install instructions. It looks like these will be addressed by [this PR](https://github.com/artefactual/atom-docs/pull/163) but I thought I'd highlight my observations in relation to the existing releases' docs.

The docs instruct to add the nodesource repo as follows:
```
curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
```
However, in Ubuntu 16.04 and 18.04, `npm` isn't actually installed from the added repo; it comes from the Ubuntu repositories:
```
vagrant@ubuntu-bionic:~$ apt-cache policy npm
npm:
  Installed: 3.5.2-0ubuntu4
  Candidate: 3.5.2-0ubuntu4
  Version table:
 *** 3.5.2-0ubuntu4 500
        500 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
        100 /var/lib/dpkg/status
```
Same for `nodejs` on Ubuntu 18.04 due to the Ubuntu repo version being newer than the one from the added repo:

```
vagrant@ubuntu-bionic:~$ apt-cache policy nodejs
nodejs:
  Installed: 8.10.0~dfsg-2ubuntu0.4
  Candidate: 8.10.0~dfsg-2ubuntu0.4
  Version table:
 *** 8.10.0~dfsg-2ubuntu0.4 500
        500 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages
        100 /var/lib/dpkg/status
     8.10.0~dfsg-2ubuntu0.2 500
        500 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages
     8.10.0~dfsg-2 500
        500 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
     6.14.1-1nodesource1 500
        500 https://deb.nodesource.com/node_6.x bionic/main amd64 Packages
```

With regard to `npm`, AFAICT the Ubuntu repo version is installed because the added nodesource repo doesn't include an `npm` package, although the output from the `curl` command to add the repo suggests otherwise:
```
## Run `sudo apt-get install -y nodejs` to install Node.js 6.x LTS Boron and npm
```
The [Packages file](https://deb.nodesource.com/node_6.x/dists/bionic/main/binary-amd64/Packages) also suggests this is the case:
```
Provides: nodejs-dev, nodejs-legacy, npm
````
Perhaps this explains why the AtoM installation instructions for 16.04 instruct to install `nodejs` and `make` only, but then for 18.04 instruct to install `npm` as well as `nodejs` and `make`. In other words, the author encountered the same issue where `npm` wasn't available after installing `nodejs`. It looks like we're [not the only ones](https://github.com/nodesource/distributions/issues/1027) to have noticed this.

Another observation I've made is that on 16.04, 18.04 & 20.04, it doesn't appear to matter which version of `npm` is installed if ensuring that `less@<2.0.0` is all that you care about. And also an explicit `nodejs` & `make` install isn't required. On a clean VM type:
```
sudo apt update && sudo apt install -y npm
sudo npm install -g "less@<2.0.0"
lessc --version # returns 1.7.5
cd /tmp
wget https://storage.accesstomemory.org/releases/atom-2.6.4.tar.gz && tar -xvf atom-2.6.4.tar.gz
cd /tmp/atom-2.6.4/plugins/arDominionPlugin
make -C ${PWD}
# make: Entering directory '/tmp/atom-2.6.4/plugins/arDominionPlugin'
# Running LESS compiler...
# lessc --compress --relative-urls css/main.less > css/main.css
# make: Leaving directory '/tmp/atom-2.6.4/plugins/arDominionPlugin'
```
The less compiler successfully compiled the `arDominion` theme.

Am I missing something, e.g. are there other use-cases for `nodejs` & `npm` where the nodesource repo version is required?
According to the PR mentioned earlier, using the Ubuntu repo version of `npm` will be in the `2.7` instructions, though probably worth streamlining the older releases' install instructions?

**Comment by jraddaoui** [comment]
Hi @Jimadine,

Thanks for the analysis and observations. There are no other use case in production that require `nodejs` nor 
`npm` and I'm not sure why we forced that version at some point. So, as you said, installing only the default `npm` version should be enough. We'll revisit the PR you mention and add a similar change to the current installation docs for 2.6.x.

**Comment by jraddaoui** [comment]
Thanks again @Jimadine,

We're going to address this in 2.6 with the following pull request until we implement a better solution for 2.7 (see https://github.com/artefactual/atom-docs/pull/163#issuecomment-860957107 for more information).

https://github.com/artefactual/atom-docs/pull/183

In the new commit, we're removing the `nodejs` installation but keeping the `make` install even if it's not needed in Ubuntu 18.04, just in case someone is following the doc in a different OS, as it may solve some minor headaches.

**Comment by jraddaoui** [comment]
On second thought, we are planing a new version of the existing themes using Bootstrap 5, which may not come in 2.7. Considering the deployment changes the suggestion and PR above will require, we probably want to wait to do that alongside the B5 upgrade. Considering that the remaining warnings are deprecated options running the Less compiler and that we are using a pinned version already, it shouldn't be a mayor concern.

**Comment by jraddaoui** [comment]
Sorry for the back and forward @Jimadine, finally addressed in 2.6 (9bbcb2a) and 2.7 (a885d83) with similar fixes. Thanks!
